templates/index.html

```diff
@@ -157,20 +159,20 @@ <h3 class="p-heading--4">OpenStack has tens of components and options. Not every
        </div>
     </div>  
     <div class="row u-equal-height">
-      <div class="col-3">
-        <ul class="p-list u-no-margin--bottom__medium">
+      <div class="col-3 col-medium-2">
+        <ul class="p-list u-no-margin--bottom">
           <li class="p-list__item is-ticked" style="border-bottom: 0">Keystone</li>
           <li class="p-list__item is-ticked">Horizon</li>
           <li class="p-list__item is-ticked last-item">Glance</li>
         </ul>
       </div>
-      <div class="col-3">
-        <ul class="p-list u-no-margin--bottom__medium">
```

Comment:
This `.u-no-margin--bottom__medium` class is in this project's styles.scss, can we remove that? I don't think it appears elsewhere.